common/components/filter/filter.yaml

```diff
@@ -25,10 +25,15 @@ params:
             type: string
             required: false
             description: the content of the small text inside the link
+          - name: classes
+            type: string
+            required: false
+            description: the css class applied to the container
 
 examples:
   - name: default
     data:
+      classes: 'app-filter-stacked'
```

Comment:
You might want to split this out into a separate example, as if we start to generate examples using these configurations, the stacked variation will show as the "default" version, which I don't think we want?

In other components we have a [test to pass in the classes attribute](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/components/card/template.test.js#L73-L83) to ensure it works as expected and don't actually worry about creating an "example".

Reply:
Ok, I'll keep that in mind. In this case, the attribute doesn't generate any different behaviour, apart from adding the class itself, so I guess it's fine (-ish).

Reply:
Oh yes, it's missing the double hyphen. Again, it threw me because it had the name of the modifier you created.